src/tests/testClpMap.cc

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+#include "SquidConfig.h"
+#include "testClpMap.h"
+#include "unitTestMain.h"
+
+#include <ctime>
+
+CPPUNIT_TEST_SUITE_REGISTRATION( testClpMap );
+
+class SquidConfig Config;
+
+void
+testClpMap::addSequenceOfElementsToMap(TestMap &m, int count, int startWith, TestMap::Ttl ttl)
+{
+    for (auto j = startWith; j < startWith + count; ++j)
+    {
+        CPPUNIT_ASSERT_EQUAL(true, m.add(std::to_string(j), j, ttl));
```

Comment:
To simplify (no value in printing booleans in this case):

```suggestion
        CPPUNIT_ASSERT(m.add(std::to_string(j), j, ttl));
```